### PR TITLE
Static main page

### PR DESCRIPTION
**Pull request by Logopeddd**

### CODE REVIEW

File: src/main/app/App.js [hunk 1]
```diff
@@ -1,21 +1,23 @@
-import React, { Component } from 'react';
-import logo from '../../assets/logo.svg';
+import React from 'react';
+import { App } from 'grommet';
```
Comment (aaneitchik) [inline src/main/app/App.js]: I would suggest using 'as' syntax and write something like 'GrommetApp' to get rid of _App

File: src/components/tiles/tiles.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+const TilesComponent = () => (
+  <Tiles
+    pad={{ horizontal: 'large' }}
+    selectable
+    flush={false}
+    fill
+    colorIndex="light-2"
+  >
+    <Tile>
```
Comment (aaneitchik) [inline src/components/tiles/tiles.js]: Create reusable components for Tile

File: src/main/app/App.js [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
+import '../../../node_modules/grommet-css';
```
Comment (aaneitchik) [inline src/main/app/App.js]: Just `import 'grommet-css';`

**Review by aaneitchik — CHANGES_REQUESTED** [review]
nice :)
please remove package-lock.json from .gitignore, and add prettier rules to ESLint

File: src/main/app/App.js (hunk 3 above)
Comment (Logopeddd) [reply src/main/app/App.js]: nope, it's necessary to import  /node_modules/grommet-css
https://www.npmjs.com/package/grommet-css

Comment (aaneitchik) [reply src/main/app/App.js]: are you sure? I tried using just 'grommet-css', and it seems like everything is ok

Comment (Logopeddd) [reply src/main/app/App.js]: try npm install anew and check, refresh page several times
could be all seems like everything is ok cause you tried it after installing 'grommet'
image

**Review by aaneitchik — APPROVED** [review]
👍

**Comment by Logopeddd** [comment]
mian_page-tile XD

**Review by aaneitchik — APPROVED** [review]

**Pull request merged**